src/embed_tests/TestFinalizer.cs

```diff
@@ -47,6 +48,7 @@ private static bool FullGCCollect()
         }
 
         [Test]
+        [Ignore("Ignore temporarily")]
```

Comment:
Please create a work item for newly disabled tests and assign to me.

Reply:
Done, #1309.